### PR TITLE
Fix parameter name and structure for radius server setting for VpnGat…

### PR DESCRIPTION
**Pull request by ritwikbasu**
…eways.

Fix parameter name and structure for radius server setting for
VpnGateways.

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

The Rest-API-Spec PR for this change is here:
https://github.com/Azure/azure-rest-api-specs/pull/1484

### CODE REVIEW

File: src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGateway.cs [hunk 1]
```diff
@@ -63,19 +63,14 @@ public VirtualNetworkGateway()
...
         /// VirtualNetworkGateway resource. Possible values are: 'Updating',
         /// 'Deleting', and 'Failed'.</param>
         /// <param name="etag">Gets a unique read-only string that changes
         /// whenever the resource is updated.</param>
-        public VirtualNetworkGateway(string id = default(string), string name = default(string), string type = default(string), string location = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), IList<VirtualNetworkGatewayIPConfiguration> ipConfigurations = default(IList<VirtualNetworkGatewayIPConfiguration>), string gatewayType = default(string), string vpnType = default(string), bool? enableBgp = default(bool?), bool? activeActive = default(bool?), SubResource gatewayDefaultSite = default(SubResource), VirtualNetworkGatewaySku sku = default(VirtualNetworkGatewaySku), VpnClientConfiguration vpnClientConfiguration = default(VpnClientConfiguration), BgpSettings bgpSettings = default(BgpSettings), string radiusServer = default(string), string radiusSecret = default(string), string resourceGuid = default(string), string provisioningState = default(string), string etag = default(string))
```
Comment (shahabhijeet) [inline src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGateway.cs]: @ritwikbasu so anyone who use to provide radiusServer info will be broken during constructing VirtualNetworkGateway?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
@ritwikbasu if this is a breaking change.
Bump up the major version.
Also where are the tests for this change?

File: src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGateway.cs (hunk 1 above)
Comment (ritwikbasu) [inline src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGateway.cs]: This feature is not exposed to customers. It has been deployed in NRP but is protected by a feature flag. This change ensures that the contract matches NRP and the documentation that will go out to the customer once the feature is released publicly.

The correct behavior is that radius settings should go as part of VpnClientConfiguration object when the feature is released. That is also the behavior in ARM/NRP now. This is a bugfix on the previous iteration which exposed radius address.

Comment (ritwikbasu) [inline src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGateway.cs]: This is not a breaking change - this is a bugfix.

The tests are compiled as part of the powershell release. This specific change requires pre-setup that is already available in powershell.

**Pull request closed without merging**